src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1526,7 +1518,8 @@ internal bool IsSubmissionSyntaxTree(SyntaxTree tree)
         /// <summary>
         /// Global imports (including those from previous submissions, if there are any).
         /// </summary>
-        internal ImmutableArray<NamespaceOrTypeAndUsingDirective> GlobalImports => _globalImports.Value;
+        internal ImmutableArray<NamespaceOrTypeAndUsingDirective> GlobalImports
+            => InterlockedOperations.Initialize(ref _globalImports, () => BindGlobalImports());
```

Comment:
```suggestion
            => InterlockedOperations.Initialize(ref _globalImports, static () => BindGlobalImports());
```

Reply:
That doesn't work for me as the method is not static.

Reply:
But `static self => self.BindGlobalImports()` would work similar to other cases?

Reply:
Whoops, yes, should use the args overload here too